docs: add known issue for cni folder issue (#5596)
spectrocloud/librarium: docs/docs-content/troubleshooting/edge.md

## Scenario - Clusters with Cilium and RKE2 Experiences Kubernetes Upgrade Failure

> When you upgrade your cluster from RKE2 1.29 to 1.30 and your cluster uses the Cilium Container Network Interface(CNI),

Author: ```suggestion
When you upgrade your cluster from RKE2 1.29 to 1.30 and your cluster uses the Cilium Container Network Interface (CNI),
```

   ```shell
   systemctl restart kubelet
   ```

## Scenario - Agent Mode Deployments CNI Folder Permission Issues

> Agent mode clusters that use PKX-E as the Kubernetes layer have the contents of the `/opt/cni/bin` folder set in
> correctly. This prevents the CNI from operating.

Reviewer: ```suggestion
Agent mode clusters that use PKX-E as the Kubernetes layer have the contents of the `/opt/cni/bin` folder set 
incorrectly. This prevents the CNI from operating.
```

> correctly. This prevents the CNI from operating.

Reviewer 2: This only affects CNI providers that **do not** run as root such as Cilium. For example, using Calico CNI does not require this workaround.

### Debug Steps

1. Log in to [Palette](https://console.spectrocloud.com).

2. Navigate to the left **Main Menu** and click on **Profiles**.

3. Click on the profile used by your agent mode cluster.

4. In the OS pack of your agent mode cluster profile, configure the following cloud-init stages. The same commands are

>    run in multiple stages to ensure that they take effect.

Author: ```suggestion
   executed in multiple stages to ensure that they take effect.
```
